### ISSUE TITLE
How to adapt the framework to support recurrent policy?

### ISSUE THREAD

**Issue by yangysc** [facebookresearch/ReAgent#137]
Dear all, 
         It's nice to have such a powerful framework. I'm wondering if the recurrent policy could be supported, since the LSTM policy is essential for solving some partial MDP tasks.

Thanks
Shanchao Yang

**Comment by czxttkl** [comment]
We haven't had one. But there is a closely-related prototype in `state_embed_gym.py`:
"""
This file shows an example of using embedded states to feed to RL models in
partially observable environments (POMDPs). Embedded states are generated by a world
model which learns how to encode past n observations into a low-dimension
vector. Embedded states improve performance in POMDPs compared to just using
one-step observations as states because  encode more historical information.
"""

Basically, we first generate the embeddings of past-n-steps observations then apply RL models on the **embedded state**.

**Comment by yangysc** [comment]
Thanks for your kind response. I'll try it after my PhD application.
